src/main/kotlin/uk/gov/justice/digital/hmpps/makerecalldecisionapi/domain/makerecalldecisions/UpdateRecommendationResponse.kt

```diff
@@ -0,0 +1,11 @@
+package uk.gov.justice.digital.hmpps.makerecalldecisionapi.domain.makerecalldecisions
+
+import uk.gov.justice.digital.hmpps.makerecalldecisionapi.jpa.entity.Active
+
+data class UpdateRecommendationResponse(
```

Comment:
Can we not just have a 'RecommendationResponse' or similar which would allow us to remove the 'CreateRecommendationResponse' object which is basically just a slimmed down version of this?

Reply:
Done :)